awx_collection/plugins/modules/tower_credential_type.py

```diff
@@ -64,12 +64,11 @@
       default: "present"
       choices: ["present", "absent"]
       type: str
-    validate_certs:
+    tower_oauthtoken:
```

Comment:
Adding the `tower_oauthtoken` docstring to each converted module until everything has changed over, at which point this docstring can just be added to the `awx_collection/plugins/doc_fragments/auth.py` file instead.

Reply:
I'm not sure we want to remove the "validate_certs" option here, that is to disable validating the tower ssl certs, it seems that the oauth token option does not replace this but is instead supplementary

Reply:
The validate_certs option can be removed from the individual modules because it is defined in tower_api.py line 42.